src/Image.cc

```diff
@@ -268,6 +268,12 @@ NAN_SETTER(Image::SetOnload) {
   if (value->IsFunction()) {
     Image *img = Nan::ObjectWrap::Unwrap<Image>(info.This());
     img->onload = new Nan::Callback(value.As<Function>());
+  } else if (value->IsNull()) {
+    Image *img = Nan::ObjectWrap::Unwrap<Image>(info.This());
+    if (img->onload) {
+      delete img->onload;
+    }
+    img->onload = NULL;
```

Comment:
I was using NULL here as it is used when initialising a new Image :smiling_imp:.

Shouldn't we use nullptr across all the project then?